modules/partition-distribution/src/main/java/org/apache/ignite/internal/partitiondistribution/AssignmentsChain.java

```diff
@@ -30,15 +33,19 @@
  * Contains the chain of changed assignments.
  */
 public class AssignmentsChain {
+
+    private static final long DEFAULT_CONF_TERM = -1;
```

Comment:
Why do we need defaults for term and index? In other words in which cases are we going to use them?

Reply:
I have plans to  set -1, -1 in `TableManager#writeTableAssignmentsToMetastore` in case when we initially set stable assignments along with chains. We do not know the index and term on which raft has applied cfg at the very beginning, when we initialise stable.

Reply:
@sanpwc